public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -496,6 +495,12 @@ export class CloudWatchDatasource extends DataSourceApi<CloudWatchQuery, CloudWa
       );
     }
 
+    const resultsToDataFrames = (val: any): DataFrame[] => {
+      //???????????????????????????????????
+      console.log('CONVERT', val);
+      return [];
+    };
+
     return from(this.awsRequest(TSDB_QUERY_ENDPOINT, requestParams)).pipe(
       map(response => resultsToDataFrames(response)),
```

Comment:
@kaydelaney @aocenas -- any idea what is happening here?  I am super suspicious that the call to `resultsToDataFrames` was correct as it would only parse base64 encoded arrow frames...

This PR moves the kinda wonky format to the runtime package.  But I suspect this was always returning an empty array (regardless or the response)